recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
```

Comment:
I've used quotes here in my other recipes, and I recall seeing that it's recommended to use them here.